scipy/stats/tests/test_distributions.py

```diff
@@ -5526,3 +5420,19 @@ def test_support_broadcasting_gh13294_regression():
     assert_equal(b2, ex_b2)
     assert a2.shape == ex_a2.shape
     assert b2.shape == ex_b2.shape
+
+
+def test_distr_params_lists():
+    # distribution objects are extra distributions added in
+    # test_discrete_basic. All other distributions are strings (names)
+    # and so we only choose those to compare whether both lists match.
+    discrete_distnames = {name for name, _ in distdiscrete
+                          if isinstance(name, str)}
+    invdiscrete_distnames = {name for name, _ in invdistdiscrete
+                             if isinstance(name, str)}
+    assert discrete_distnames == invdiscrete_distnames
+
+    cont_distnames = {name for name, _ in distcont if isinstance(name, str)}
+    invcont_distnames = {name for name, _ in invdistcont
+                         if isinstance(name, str)}
+    assert cont_distnames == invcont_distnames
```

Comment:
```suggestion
    discrete_distnames = {name for name, _ in distdiscrete}
    invdiscrete_distnames = {name for name, _ in invdistdiscrete}
    assert discrete_distnames == invdiscrete_distnames

    cont_distnames = {name for name, _ in distcont}
    invcont_distnames = {name for name, _ in invdistcont}
    assert cont_distnames == invcont_distnames
```
Those aren't needed now, right?

Reply:
Actually, `distdiscrete` is imported from the `test_discrete_basic` where a distribution object is added. Other lists don't have distribution objects, so, we will have to keep the condition for `distdiscrete` and can remove from other lists. I will do this in the next commit.

Reply:
Well, what if we just made a copy of the list in `test_discrete_basic`? Do you think it was really intended for it to modify this list?

Reply:
> Wait, what if we just made a copy of the list in test_discrete_basic?

Can do that but I think it will be an unrelated change in this PR. 

> Do you think it was really intended for it to modify this list?

I don't know how it got added so can't say whether it was intended or not.

Reply:
OK.